packages/core/src/types.ts

```diff
@@ -497,6 +500,18 @@ export type Media = {
   contentType?: string;
 };
 
+/**
+ * Represents a flexible button configuration
+ */
+export type Button = {
```

Comment:
Maybe we could move it to plugin-telegram if it's only used for Telegram? 🤔
https://github.com/elizaOS/eliza/blob/v2-develop/packages/plugin-telegram/src/types.ts